src/code.cloudfoundry.org/cni-wrapper-plugin/lib/lib.go

```diff
@@ -105,6 +107,11 @@ func LoadWrapperConfig(bytes []byte) (*WrapperConfig, error) {
 		return nil, fmt.Errorf("invalid outbound connection rate")
 	}
 
+	if n.OutConn.DryRun {
+		if !n.OutConn.Limit {
```

Comment:
[Medium] You could turn this into a oneliner:

```go
if n.OutConn.DryRun && !n.OutConn.Limit {
```

Reply:
The logic is further separated.